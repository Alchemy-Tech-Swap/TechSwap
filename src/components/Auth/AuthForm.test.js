import AuthForm from './AuthForm';
import { render, screen } from '@testing-library/react';
import { MemoryRouter } from 'react-router-dom/cjs/react-router-dom.min';

test('Should render Auth Form', async () => {
  const { container } = render(
    <MemoryRouter>
      <AuthForm
        user={{
          email: '',
          errorMessage: '',
          handleSubmit: 'ƒ handleSubmit() {}',
          password: '',
          setEmail: 'ƒ bound dispatchAction() {}',
          setPassword: 'ƒ bound dispatchAction() {}',
        }}
      />
    </MemoryRouter>
  );
  await screen.findByText('Email:');
  expect(container).toMatchSnapshot();
});
