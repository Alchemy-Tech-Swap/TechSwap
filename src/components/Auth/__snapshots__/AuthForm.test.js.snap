// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render Auth Form 1`] = `
<div>
  <form
    class="form"
  >
    <h3 />
    <div
      class="form-controls"
    >
      <label>
        Email:
      </label>
      <input
        placeholder="Enter Valid Email Address"
        type="email"
        value=""
      />
    </div>
    <div
      class="form-controls"
    >
      <label>
        Password:
      </label>
      <input
        placeholder="Minimum 6 digits req."
        type="password"
        value=""
      />
    </div>
    <input
      class="submit"
      type="submit"
    />
  </form>
</div>
`;
